percy/driver/driverWrapper.js

```diff
@@ -12,7 +12,7 @@ class AppiumDriver {
     /* istanbul ignore else */
     // Note: else is covered here but constructor name '' couldnt cover (which is the case)
     // in real world when you get wd driver passed so need to ignore coverage on it
-    if (driver.constructor.name === 'Browser' && !Undefined(driver.getSession)) {
+    if (driver.constructor.name.includes('Browser') && !Undefined(driver.getSession)) {
```

Comment:
can u write it in the comment that what will be the `driver.constructor.name` that u r going to support for wdio

Reply:
resolved